docs/multicast-guide.md

```diff
@@ -165,3 +166,10 @@ is not routed correctly:
 
 1. Node kernel version under 5.4
 2. Node network doesn't support IGMP snooping
+
+### Antrea FlexibleIPAM
+
+The configuration option `multicastInterfaces` is not supported with
+[Antrea FlexibleIPAM](antrea-ipam.md#antrea-flexible-ipam).
+When Antrea FlexibleIPAM is enabled, multicast packets are forwarded
```

Comment:
Do not add a new line between sentences. Just wrap the lines at 80.

Reply:
Updated. The wrappings of the multicast guide doc have been fixed.